File: frontend/src/constants/application_task_assignee.ts
import { Dayjs } from "dayjs";

export type application_task_assignee = {
  
  id: number;
  structure_employee_id: number;
  application_task_id: number;
  created_at?: Dayjs;
  updated_at?: Dayjs;
  created_by?: number;
  updated_by?: number;
};

File: backend/Domain/Entities/application_task_assignee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class application_task_assignee
    {
        public int id { get; set; }
		public int structure_employee_id { get; set; }
		public int employee_id { get; set; }
		public string employee_name { get; set; }
        public string employee_ocupation { get; set; }
        public string post_code { get; set; }
        public int application_task_id { get; set; }
		public DateTime? created_at { get; set; }
		public DateTime? updated_at { get; set; }
		public int? created_by { get; set; }
		public int? updated_by { get; set; }
		
    }
}